GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -463,13 +496,17 @@ extension CameraViewController {
                                               deviceOrientation: UIApplication.shared.statusBarOrientation)
         
         if giniConfiguration.multipageEnabled {
-            self.animateToControlsView(imageDocument: imageDocument) {
-                self.didPick(validatedDocuments: [imageDocument])
+            self.didPick(validatedDocuments: [imageDocument]) { error, _ in
+                guard let error = error else {
+                    self.animateToControlsView(imageDocument: imageDocument)
+                    return
+                }
+                self.showErrorDialog(for: error)
```

Comment:
Maybe this is just me, but I find this `guard` confusing. I'm used to seeing the edge cases within the guard and the successful case after it. Here, it's reversed.

Reply:
Because the 'positive' case here is to have an error, since the completion block is focused in the error.

Reply:
Ok, that makes sense. But why is the completion handler focused on the error. It also handles the successful case, doesn't it? That means that if you change the naming, it becomes a standard completion closure. 

It's not important, but I feel like it's similar to a `isNotEmpty` method, which requires a bit more mental power to understand than `isEmpty`.